srcpkgs/gdm/files/zz-gdm.sh

```diff
@@ -0,0 +1,5 @@
+# This must run after locale.sh
```

Comment:
```suggestion
## This must run after locale.sh ##
# GDM expects that the shell will respect what it sets in LANG, but login 
# shells will overwrite any locale information based on the contents of
# /etc/locale.conf. Because GDM also sets GDM_LANG to its desired language, we
# can just force the value here after locale.conf has been parsed.
```